src/OSPSuite.Core/Domain/ObjectPathKeywords.cs

```diff
@@ -61,6 +61,12 @@ public static class ObjectPathKeywords
       /// </summary>
       public static readonly string ALL_FLOATING_MOLECULES = addKeyword("ALL_FLOATING_MOLECULES");
 
+      /// <summary>
+      /// Represents the keywords allowing us to identify a neighborhood between two containers
+      /// For example CONTAINER1|/<NBH/>CONTAINER2/<NBH/> means the neighborhood between CONTAINER1 and CONTAINER2
+      /// </summary>
+      public static readonly string NBH = addKeyword("<NBH>");
```

Comment:
Should we name the constant **in the code** NEIGHBORHOOD instead of NBH?
`public static readonly string NEIGHBORHOOD = addKeyword("<NBH>");`

Reply:
there is already NEIGHBORHOOD.... so

Reply:
hmm, ok